src/Components/Card/Card.jsx

```diff
@@ -1,75 +1,93 @@
 import { useProduct } from "../../service/product_api";
 import "./Card.css";
+import { useCart } from "../../CartContext/CartContext"
 
 export function Card(props) {
-    return (
-        <div className="comp" id="card-component">
-            <div className="photo-card-container horizontal">
-                <div className="basic-card-container horizontal">
-                    <div className="photo-area horizontal">
-                        <div className="photo-comp">
-                            <div className="photo-badge horizontal">
-                                <i className="fa fa-heart"></i>
-                            </div>
-                            <img className="card-photo horizontal" src={require("./jacket.jpg")} alt="Green Jacket" loading="lazy" />
-                        </div>
-                    </div>
+  return (
+    <div className="comp" id="card-component">
+      <div className="photo-card-container horizontal">
+        <div className="basic-card-container horizontal">
+          <div className="photo-area horizontal">
+            <div className="photo-comp">
+              <div className="photo-badge horizontal">
+                <i className="fa fa-heart"></i>
+              </div>
+              <img
+                className="card-photo horizontal"
+                src={require("./jacket.jpg")}
+                alt="Green Jacket"
+                loading="lazy"
+              />
+            </div>
+          </div>
 
-                    <div className="card-footer horizontal">
-                        <div className="price">
-                            <div className="card-subheading">New Arrivals</div>
-                            <div className="small-space"></div>
-                        </div>
-                        <div className="small-space"></div>
-                        <div className="card-heading">{props.collection}</div>
-                        <div className="card-subheading detail">Get your favorite summer clothes at affordable
-                            prices</div>
-                    </div>
-                </div>
+          <div className="card-footer horizontal">
+            <div className="price">
+              <div className="card-subheading">New Arrivals</div>
+              <div className="small-space"></div>
             </div>
+            <div className="small-space"></div>
+            <div className="card-heading">{props.collection}</div>
+            <div className="card-subheading detail">
+              Get your favorite summer clothes at affordable prices
```

Comment:
Just a suggestion maintain a single file for stray strings. It becomes easy to maintain.  For example, if you want to change text in a component you won't have to go through the whole component instead you can go through constants file where you've stored all stary strings. Not necessary but good to have.

Reply:
No worries, happy to help!